build/docker/files/root/entrypoint.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -e
+
+### Verify container started with all required configuration
+
+if [ -z "$MARTI_SMTP_HOST" ]; then
+  echo >&2 '[ERROR]: specify the MARTI_SMTP_HOST environment variable (e.g. "smtp.gmail.com")'
+  exit 1
+fi
+if [ -z "$MARTI_SMTP_PORT" ]; then
+  echo >&2 '[ERROR]: specify the MARTI_SMTP_PORT environment variable (e.g. "587")'
+  exit 1
+fi
+if [ -z "$MARTI_SMTP_USERNAME" ]; then
+  echo >&2 '[ERROR]: specify the MARTI_SMTP_USERNAME environment variable (e.g. "first.last@gmail.com")'
+  exit 1
+fi
+if [ -z "$MARTI_SMTP_PASSWORD" ]; then
+  echo >&2 '[ERROR]: specify the MARTI_SMTP_PASSWORD environment variable (e.g. "my_password")'
+  exit 1
+fi
+
+### Initialize web server
+
+chown -R www-data:www-data /var/www /var/log/php
+
+### Initialize database
+
+if [ ! -d /var/lib/mysql/mysql ]; then
+  mysqld --initialize-insecure --user=root --datadir=/var/lib/mysql
+fi
+
+chown -R mysql:mysql /var/lib/mysql
+
+if [ ! -d "/var/lib/mysql/$MARTI_DB_NAME" ]; then
+  service mysql start
```

Comment:
The parent image of my Docker container does not auto start any services controlled by `supervisord` (of which MySQL is one).  So it is guaranteed that MySQL is not running at this point (`supervisord` is not started until the last line in the `entrypoint.sh` script).